pkg/postgres/pgtest/postgres.go

```diff
@@ -22,6 +22,7 @@ import (
 type TestPostgres struct {
 	*pgxpool.Pool
 	database string
+	GormDB   *gorm.DB
```

Comment:
It does not seem to be necessary to add this? We do not have GormDB in central production codes.